README.md

```diff
@@ -51,6 +51,24 @@ of variables:
 In addition, you will need to ensure the bucket CORS `AllowedOrigin` settings
 are configured to allow for the domain this app runs at.
 
+## Sending Receipt Email in Production
+
+`SMTP_ADDRESS`
```

Comment:
Do we need these 5 set in every environment or just on prod?

If we don't need all 5 everywhere, what needs to be set on local?

What needs to be set on heroku when `DISABLE_ALL_EMAIL`?

Reply:
If you want to send email, you need to configure the email with these settings. Dev and test never actually send email (even if you set those). Dev writes to a file and test writes to an in memory location (an array). Can you re-read this section and let me know if you have specific suggestions of what to change as I thought I covered that.

Reply:
The confusion I'm having is that (I think) four of these only need to be set if you're using SMTP, but one of them needs to be set even if you're not because you'll end up with a runtime error. (And also the thesis_admin_email setting is documented in a section whose header includes "...in production", but you have to also set it in other environments.)

I think I would like more clarity as to when you set what. `Sending Receipt Email in Production` vs `Handling email in other environments` (which might then have subsections `to test email` and `to disable email`?)

There's a lot of moving parts here and I want to be certain I know how to move them to accomplish different scenarios.

Reply:
Thx for the clarification on the cover. I think I can make words that explain it better. 👍

Reply:
I've put in a very small change to clarify the dev requirement for email in the dev env section. I also opened https://mitlibraries.atlassian.net/browse/TI-69 as I think there is room for significant improvement in grouping the information which ultimately led to this concern (not being consistent in how we are documenting settings / features).